tests/keras_tests/feature_networks_tests/feature_networks/second_moment_correction_test.py

```diff
@@ -247,7 +246,6 @@ def run_test(self, experimental_facade=False, experimental_exporter=False):
     def prepare_graph(self,
                       in_model: Model,
                       representative_data_gen: Callable,
-                      n_iter: int = 500,
```

Comment:
If you remove the n_iter from here, where is the 500 iterations set for the tests? (maybe I just can't find it in all the files and changes... or is it just not necessary anymore?)

Reply:
set num_calibration_iter=100 in BaseSecondMomentTest. apparently 100 is enough.